client/my-sites/preview/style.scss

```diff
@@ -1,16 +0,0 @@
-.main.preview {
-	height: 100%;
-	max-width: none;
-
-	.web-preview__toolbar {
-		position: sticky;
-		z-index: 1;
-		top: var(--masterbar-height);
-	}
-}
-
-.layout.is-section-preview .layout__content {
```

Comment:
There's a few more references to `is-section-preview` in `client/layout/style.scss`. Can those be removed as well?

https://github.com/Automattic/wp-calypso/blob/84c643a7596e49b50422a980814560f539693338/client/layout/style.scss#L110-L115